docs/provision_hypershift_clusters_by_mce.md

```diff
@@ -101,6 +101,8 @@ stringData:
   aws_access_key_id:      # Value, required
 ```
 
+The `ssh-publickey` and `ssh-privatekey`, if provided, are used to access the worker nodes of the hosted cluster. The SSH Key in the `HostedCluster.Spec` or `HostedDeployment.Spec` takes procedence if they are provided.
```

Comment:
precedence (spelling), and over what?  We should mention it takes precedence over "Cloud Provider"